Net/src/SocketImpl.cpp

```diff
@@ -1372,12 +1372,12 @@ void SocketImpl::error(int code, const std::string& arg)
 		throw IOException(NumberFormatter::format(code), arg, code);
 	}
 }
-
+#ifdef POCO_HAVE_SENDFILE
```

Comment:
Still why won't we just implement sendFile so that it can be used with every platforms? It would just use sendfile / sendfile64 / TransmitFile when those are available. Then users can just always use our sendFile without thinking platforms. This is probably what they want anyway if they want to make multiplatform application. It is Poco's job to hide these kind of things.

We example also do not have seperete sendFile and transmitFile functions.